index.html

```diff
@@ -16,10 +16,14 @@
 <body>
     <div id='map'></div>
     <div class='map-overlay' id='features'>
+        <h3>Can my municipality supply itself with electricity from sun and wind?</h3>
         <h2>Renewable electricity autarky</h2>
         <div id='pd'>
-            <p>Hover over an area for details.<br>
-            Zoom to switch administrative levels.</p>
+            <p>
+                Hover over an area for details<br>
+                Zoom to switch administrative levels.<br>
+                Click on  <i class="fas fa-info-circle"></i> for more information.<br>
```

Comment:
I think it's confusing that this info symbol is black. It should be green as well.

Reply:
finde ich nicht. muss du wenn dann selbst ändern, kann ich nicht so einfach